### PR TITLE
Backport fix displaying inner exceptions

### PR DESCRIPTION
**Pull request by Evangelink**
Backport #3920 to rel/3.6

### CODE REVIEW

**Comment by thomhurst** [comment]
Can we get this in please? 😄

**Comment by nohwnd** [comment]
I tried to cherry-pick the backported change into this branch and there were several problems, this will need to wait for Amaury to return (I think tomorrow).

File: test/UnitTests/Microsoft.Testing.Platform.UnitTests/OutputDevice/Terminal/TerminalTestReporterTests.cs [hunk 1]
```diff
@@ -98,9 +98,8 @@ public void OutputFormattingIsCorrect()
                 ABC
               Actual
                 DEF
-            ␛[m␛[91m  Stack Trace:
-                ␛[90mat ␛[m␛[91mFailingTest()␛[90m in ␛[90m␛]8;;file:///{folderLink}codefile.cs␛\{folder}codefile.cs:10␛]8;;␛\␛[m
-
+            ␛[m␛[90m    ␛[90mat ␛[m␛[91mFailingTest()␛[90m in ␛[90m␛]8;;file:///C:/work/codefile.cs␛\C:\work\codefile.cs:10␛]8;;␛\␛[m
```
Comment (Evangelink) [inline test/UnitTests/Microsoft.Testing.Platform.UnitTests/OutputDevice/Terminal/TerminalTestReporterTests.cs]:
```suggestion
            ␛[m␛[90m    ␛[90mat ␛[m␛[91mFailingTest()␛[90m in ␛[90m␛]8;;file:///{folderLink}codefile.cs␛\{folderLink}codefile.cs:10␛]8;;␛\␛[m
```

**Review by nohwnd — DISMISSED** [review]

File: test/UnitTests/Microsoft.Testing.Platform.UnitTests/OutputDevice/Terminal/TerminalTestReporterTests.cs [hunk 2]
```diff
@@ -98,9 +98,8 @@ public void OutputFormattingIsCorrect()
                 ABC
               Actual
                 DEF
-            ␛[m␛[91m  Stack Trace:
-                ␛[90mat ␛[m␛[91mFailingTest()␛[90m in ␛[90m␛]8;;file:///{folderLink}codefile.cs␛\{folder}codefile.cs:10␛]8;;␛\␛[m
-
+            ␛[m␛[90m    ␛[90mat ␛[m␛[91mFailingTest()␛[90m in ␛[90m␛]8;;file:///{folderLink}codefile.cs␛\{folderLink}codefile.cs:10␛]8;;␛\␛[m
```
Comment (Evangelink) [inline test/UnitTests/Microsoft.Testing.Platform.UnitTests/OutputDevice/Terminal/TerminalTestReporterTests.cs]:
```suggestion
            ␛[m␛[90m    ␛[90mat ␛[m␛[91mFailingTest()␛[90m in ␛[90m␛]8;;file:///{folderLink}codefile.cs␛\{folder}codefile.cs:10␛]8;;␛\␛[m
```

**Pull request merged**